README.md

```diff
@@ -1 +1,39 @@
-# silverstripe-tx-translator
+# silverstripe tx translator
+
+Module to assist with transifex integration. Works with Silverstripe 4.12+ and 5.0+.
+
+## Operations
+
+- Pull latest translations from transifex and merge them into yml/json/js translation files
+- Run i18nTextCollectorTask on a local website
+- Push updated source strings to transifex (optional)
+- Create github pull-requests with file changes
+
+## Requirements
+
+Transifex cli client (tx) - [https://developers.transifex.com/docs/cli](https://developers.transifex.com/docs/cli) configured API key from transifex.
+
+You must use the go version at least version 1.6+ and not the old python version.
+
+## Usage
+
+```
+cd /path/to/my-local-site
+composer install silverstripe/tx-translator
+TX_SITE=my-local-site.test TX_PUSH=1 php vendor/silverstripe/tx-translator/script/translate.php
```

Comment:
```suggestion
# if you install this as a dependency of your silverstripe project, `path/to/tx-translator` will be `vendor/silverstripe/tx-translator`
TX_SITE=my-local-site.test TX_PUSH=1 php path/to/tx-translator/script/translate.php
```

Reply:
Requires `composer install` - see comment below in autoloader section

Reply:
I disagree but apparently I can't convince you so I'll just resolve the conversation.